index.js

```diff
@@ -56,10 +56,19 @@ module.exports = function prettyFactory (options) {
   const errorLikeObjectKeys = opts.errorLikeObjectKeys
   const errorProps = opts.errorProps.split(',')
 
-  if (opts.dateFormat.length > 0 && opts.dateFormat !== CONSTANTS.DATE_FORMAT) {
+  if (opts.localTime) {
+    if (opts.localTime.length > 0) {
+      opts.dateFormat = opts.localTime
```

Comment:
With this PR, `opts.dateFormat` is obsolete. It should be made into a local variable only and removed from the list of options.